lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb

```diff
@@ -24,6 +24,15 @@ def space_type_get_standards_data(space_type)
       space_type_properties = {}
     end
 
+    # This merges the ventilation, exhaust and lighting controls
+    data2007 = @std_2007.space_type_get_standards_data(space_type)
+    if data2007.nil?
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.SpaceType', "Space type properties from ASHRAE 90.1-2007 lookup failed")
+    else
+      space_type_properties = data2007.merge(space_type_properties)
+      space_type_properties['space_type_2007'] = data2007['space_type']
+    end
```

Comment:
And the magic happens here... Because we never define any ventilation, exhaust, or lighting control, we do NOT have that data in our JSONs.

So we load the actual Space Type info from 2007, we merge ours into this one (meaning it replaces all values it has which are per primary building type and "Whole Building", and the ventilation, exhaust is still from 2007 space type level)

Reply:
okay, would we need to check that the new space type definition is complete?  ie, not missing something that is already in the base?